e2e/puppeteer-custom-environment.ts

```diff
@@ -23,7 +23,7 @@ class PuppeteerCustomEnvironment extends PuppeteerEnvironment {
     case 'test_fn_failure':
       console.log('test failed: \t', event.test.name);
       if (state.currentlyRunningTest.invocations > retryTimes) {
-        const testName = state.currentlyRunningTest.name.replace(/\s/g, ''); // remove whitespaces
+        const testName = fp.startCase(state.currentlyRunningTest.name).replace(/(^\w|\s)/g, '');
```

Comment:
make screenshot filename clearer. e.g.
`Test "User can create a workspace using all inputs" failed. Saved screenshot logs/screenshot/serCanCreateAWorkspaceUsingAllInputs_1589299608904.png.`

Reply:
"screenshot/serCanCreateAWorkspaceUsingAllInputs" did you lose a U?